tests/Aspire.EndToEnd.Tests/Aspire.EndToEnd.Tests.csproj

```diff
@@ -21,8 +21,8 @@
     <DefineConstants Condition="'$(_BuildForTestsRunningOutsideOfRepo)' == 'true'">BUILD_FOR_TESTS_RUNNING_OUTSIDE_OF_REPO;$(DefineConstants)</DefineConstants>
 
     <XunitRunnerJson>xunit.runner.json</XunitRunnerJson>
-    <RunSettingsFilePath>$(MSBuildThisFileDirectory).runsettings</RunSettingsFilePath>
     <TestArchiveTestsDir>$(TestArchiveTestsDirForEndToEndTests)</TestArchiveTestsDir>
+    <TestingPlatformCommandLineArguments>$(TestingPlatformCommandLineArguments) --show-live-output on</TestingPlatformCommandLineArguments>
```

Comment:
NOTE: `TestingPlatformCommandLineArguments` works only when doing `dotnet test`. But if the build happens on one machine, and test is run on another machine on the already-built assembly, this won't work.

@radical @danmoseley Is that okay for you?
I don't know if this was done for easier local debugging, or is it meant for CI scenarios, and is it relevant for both helix and non-helix?

One way this could be addressed is if xUnit supported testconfig.json: https://github.com/xunit/xunit/issues/3247

Reply:
Or, we could always `--show-live-output` in all places where we invoke tests, and have consistency for all projects rather than being specified for only 3 projects currently.

Reply:
We did it for specific projects only because they would have crashes, or take too long and it would help to watch the running output. I think it might be too verbose for other tests. We can start with no `--show-live-output` at all for now, and we can turn it on if we hit a need for it again.